### DISCUSSION TITLE
Are old CUDA (pre 5.0) GPU's supported?

### DISCUSSION THREAD

**Discussion by rewardcoin-network** [tmrlvi/kaspa-miner#2] [Q&A] [answered]
Any idea if this will work with an older nvidia K80?

**Comment by tmrlvi** [comment, marked as answer]
The plan is to support also old cards (since there is not memory issue)
The plan to have it ready within a week or two, but the current priority is bug fixing

**Reply by tmrlvi** [reply]
K80 should now be supported
